src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -52,24 +52,13 @@
                    for="name"
                    i18n>Storage Backend</label>
             <div class="cd-col-form-input">
-              <select class="form-select"
-                      formControlName="name"
-                      name="name"
-                      id="name"
-                      (change)="fsalChangeHandler()">
-                <option *ngIf="allFsals === null"
-                        value=""
-                        i18n>Loading...</option>
-                <option *ngIf="allFsals !== null && allFsals.length === 0"
-                        value=""
-                        i18n>-- No data pools available --</option>
-                <option *ngIf="allFsals !== null && allFsals.length > 0"
-                        value=""
-                        i18n>-- Select the storage backend --</option>
-                <option *ngFor="let fsal of allFsals"
-                        [value]="fsal.value"
-                        [disabled]="fsal.disabled">{{ fsal.descr }}</option>
-              </select>
```

Comment:
why are we removing this?

Reply:
if this is removing the storage back-end selector completely, instead of that when routing through cephfs, we can just select the `cephfs` as storage back-end and if rgw then the RGW can be selected?

Reply:
If user clicked nfs under Object, it should be always Object and he should not select cephfs from object.
I have changed it to readonly input therefore

Reply:
I ahve completely removed this section as its redundant.